pages/analysis.md

```diff
@@ -1,15 +1,132 @@
+(left image, black lines) and the vessels that will eventually be affected 
+by privateers or pirates (right image, purple lines). Here it can be viewed 
+how little effect ships captured by these two parties could have had on the 
+slave trade. There is still no change in ships and their actions to this point. 
+On this image the ships left their home ports and headed to the ports where 
+they will be loading slaves onboard. 
+
+---
+
+![image]({{ site.baseurl}}/assets/images/3.%20To%20slave%20drop%20off%20point.png)
+
+The slaves are picked up in Africa and journeyed to the Caribbean 
+and West Indies region. Until this point, there has been little to no change
+in pattern comparing ships never captured to those that were eventually affected 
+by privateers or pirates. This image represents the path the vessels took to reach
+the slave market ports. The change in behavior occurs in this image. The left image 
+shows the mass amount of ships porting into the large slave market hubs. The right 
+image shows slaves being dropped off at similar areas but in more sporadic ports 
+compared to themasses. The interdiction of ships seems to have occurred closer to
+the Caribbean and West Indies region.  	
+	
+---
+
+![image]({{ site.baseurl}}/assets/images/4.%20Main%20Ports%20of%20Drop%20off.png)
+
+A clear visual of the ports where the slaves were dropped off is available in this
+image. When comparing both photos of the pirated and non-pirated drop off ports, it 
+seems there are no significant changes. Looking a little more closely, one can see
+that the pirated ships are spread out more widely in the region and not largely focused 
+like the red dots. The larger dots represent the level of drop off. The larger the dots,
+the more activity took place in the port.	
+
+---
+
+![image]({{ site.baseurl}}/assets/images/5.%20Distibution%20amount%20.png)
+
+The most significant change in the region is where the pirated ships dropped off 
+their slaves for sale compared to non-pirated vessels. This image shows there was 
+a market for pirated slave ships in the West Indies. The standard slave trade ships 
+are equally distributed in the slave markets. The larger the dots, the more vessels 
+ported and dropped off their slaves. The average slave trade led the ships to Rio in 
+Brazil and La Habana, Cuba. In pirated ships, the data shows the ports were more 
+concentrated and spread out.
+
+---
+
+![image]({{ site.baseurl}}/assets/images/Captive%20by%20Pirates.png)
+
+This image is a  more clear representation of the ships and the results that followed
+once Pirates and Privateers took control of the vessels. The picture shows the changes 
+did not occur until the ship reached the region and ported in a different location than 
+non-pirated ships.
+
+---
+
+![image]({{ site.baseurl}}/assets/images/Average%20Price%20of%20Slaves%20Over%20Time.jpg)
+![image]({{ site.baseurl}}/assets/images/Pirate%20Slave%20activity%20.png)
```

Comment:
This image is interesting! I wonder if binning or grouping the years might help give a slightly less cluttered picture.